crud.py

```diff
@@ -0,0 +1,41 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def get_all_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.Author).all()
```

Comment:
The `get_all_authors` function does not utilize the `skip` and `limit` parameters for pagination. You should modify the query to include `.offset(skip).limit(limit)` to properly implement pagination as per the task requirements.